tests/unit/shared/test_vic_time.py

```diff
@@ -197,18 +208,22 @@ def test_dmy_all_30_day(feb3_noon):
 @pytest.mark.skipif(nctime_unavailable, reason=nc_reason)
 def test_num2date():
 
-    dates = pd.date_range(start="1900-01-01", end="2100-12-31",
-                          freq='MS').to_pydatetime()
-
+    date_sets = (pd.date_range(start="1900-01-01", end="2100-12-31",
+                               freq='MS').to_pydatetime(),
+                 pd.date_range(start='1948-09-01 00:00:00.010000',
+                               periods=200, freq='20min').to_pydatetime(),
+                 pd.date_range(start='1948-09-02 23:59:59.09000',
+                               periods=200, freq='20min').to_pydatetime())
```

Comment:
Include some comments to make clear what you are actually testing (i.e. - why these tests)